.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: "🐞 Bug report"
+about: Something is not working as it should
+---
+#### Describe the bug
+
+ - Node.js version: 
+ - OS: 
+
+<!-- A clear and concise description of what the bug is. -->
+
+#### Actual behavior
+
+...
+
+#### Expected behavior
+
+...
```

Comment:
What are the `...` in these for?

Reply:
Just a visual thing. People would replace it with their own text. But I guess you've already known that. For me it doesn't matter if it's `...` or empty or `<!-- Provide information here. -->`. I'm good with anything :P

So, to stay consistent I'll use comments :)